### PR TITLE
object: Purge notification functionality

### PR DESCRIPTION
**Pull request by cthulhu-rider**
* based on and blocked by https://github.com/nspcc-dev/neofs-api/issues/279

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,10 +12,15 @@ Changelog for NeoFS Node
 ### Changed
 
 ### Removed
+- Object notifications incl. NATS (#2750))
```
Comment (carpawell) [inline CHANGELOG.md:15]:
```suggestion
- Object notifications incl. NATS (#2750)
```

**Review by carpawell — APPROVED** [review]
1. Do we plan supporting notifications somehow in the future on the node level? @roman-khimov 
2. Can you, please, drop that functionality in the dev-env too?

**Comment by cthulhu-rider** [comment]
> Can you, please, drop that functionality in the dev-env too?

https://github.com/nspcc-dev/neofs-dev-env/pull/305. If i forgot smth, pls report there

File: CHANGELOG.md (hunk 1 above)
Comment (cthulhu-rider) [reply CHANGELOG.md]: no smile, yeah! fixed

**Comment by roman-khimov** [comment]
1. To me it's tied to meta handling. There will be notifications eventually, but more usable ones.

**Comment by carpawell** [comment]
> https://github.com/nspcc-dev/neofs-dev-env/pull/305. If i forgot smth, pls report there

Oh, that is great, has not seen it yet.

**Pull request merged**